linkerd/http-route/src/http/filter/inject_failure.rs

```diff
@@ -73,3 +80,59 @@ impl Hash for Distribution {
         self.denominator.hash(state);
     }
 }
+
+#[cfg(feature = "proto")]
+pub mod proto {
+    use super::*;
+    use linkerd2_proxy_api::http_route as api;
+
+    #[derive(Debug, thiserror::Error)]
+    pub enum InvalidFailureResponse {
+        #[error("invalid HTTP status code: {0}")]
+        Status(#[from] http::status::InvalidStatusCode),
+
+        #[error("invalid HTTP status code: {0}")]
```

Comment:
should the message here maybe indicate that the number was too large, so that it can be distinguished from the other error? not a big deal, but just a thought.